recipes/libtiff/tiff.inc

```diff
@@ -1,24 +1,61 @@
 DESCRIPTION = "This software provides support for the Tag Image File Format (TIFF)"
+HOMEPAGE = "http://www.simplesystems.org/libtiff/"
 LICENSE = "BSD-2-Clause*"
-HOMEPAGE = "http://www.remotesensing.org/libtiff/"
 
 RECIPE_TYPES = "machine native"
+SRC_URI = "ftp://download.osgeo.org/libtiff/tiff-${PV}.tar.gz"
 
 COMPATIBLE_HOST_ARCHS = ".*linux"
 
-SRC_URI = "ftp://ftp.remotesensing.org/pub/libtiff/tiff-${PV}.tar.gz"
+inherit autotools-autoreconf c++ auto-package-utils auto-package-libs
 
-inherit autotools c++
+RECIPE_FLAGS += "tiff_jpeg tiff_lzma tiff_zlib"
```

Comment:
i don't see any configure handling of these, are they automagic ? would prefer a --disable/--without when they are disabled if that works with the provided configure